src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -81,11 +81,11 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
     <hr/>
 
     <% if (request.getSession().getAttribute("user") != null) { %>
-    <form action="/chat/<%= conversation.getTitle() %>" method="POST">
-      <input type="text" name="message">
-      <br/>
-      <button type="submit">Send</button>
-    </form>
+	    <form action="/chat/<%= conversation.getTitle() %>" method="POST">
```

Comment:
Should this be 2 spaces instead of 4?

(I honestly don't know the answer, as you can tell I'm not great at formatting jsp)

Reply:
I think two spaces past the start of the line that contains the `if` makes sense, and is consistent with the rest of the code.  (Eclipse did a goof on this particular line in this particular commit, so i had to fix it in the last commit of this PR.)